//Should fail if there is any lines of code appearing after return statement
foo = fn (x: int, y: float): float -> {
    return x + y;
    println x + y;
};
